lua/illuminate.lua

```diff
@@ -175,7 +175,9 @@ function M.next_reference(opt)
     end
     if next then
         move_cursor(next.start.line + 1, next.start.character)
-        print('['..nexti..'/'..#refs..']')
+        if navigation_message then
```

Comment:
```suggestion
        if not opt.silent then
```

Reply:
Also, did this line work before? That variable doesn't exist so it would have to be `opt.navigation_message`.

Reply:
I think I might have botched that one during the rename. I started out using `quiet` but adapted to Gitsign's naming scheme.